Libraries/oneMKL/computed_tomography/computed_tomography.cpp

```diff
@@ -288,8 +288,11 @@ sycl::event step3_ifft_2d(matrix_r &fhat,
                           const std::vector<sycl::event> &deps)
```

Comment:
I understand that verification for the sample can be tricky. Can we imagine possible errors where we can say "this is definitely incorrect" and call any "exit(1)"?

Reply:
This tomography sample is significantly more complex than the others modified herein; it may require much more time to implement a compelling verification step for it. This will be done in a separate PR. to ease the review process of these suggested changes.

Reply:
ok. Thanks for the clarification